app/src/main/java/org/oppia/app/player/audio/AudioFragmentInterface.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.app.player.audio
+
+import androidx.lifecycle.LiveData
+
+/** Interface to interact with AudioFragment from [StateFragment]. */
+interface AudioFragmentInterface {
+  /**
+   * Used to set AudioFragment's VoiceoverMappings with given stateId and contentId.
+   * If no contentId is provided, AudioFragment will use state.content.contentId as default.
```

Comment:
This is a slightly complicated contract.  Could we just always pass in the correct content ID or does the underlying fragment definitely need to subscribe to the state to retrieve it? It seems simpler to me to pass in data where possible to minimize live data observers.

Reply:
Separated setVoiceoverMapping into two separate functions to reduce complexity.